packages/portfolio-contract/test/portfolio.contract.test.ts

```diff
@@ -243,6 +246,10 @@ test('open portfolio with USDN, Aave positions', async t => {
   const { contents } = getPortfolioInfo(storagePath, common.bootstrap.storage);
   t.snapshot(contents, 'vstorage');
   t.snapshot(done.payouts, 'refund payouts');
+
+  const tree = inspectMapStore(contractBaggage);
+  delete tree['chainHub']; // 'initial baggage' test captures this
+  t.snapshot(tree, 'baggage after open with positions');
```

Comment:
It is surprising that the portfolio's contents are not included. Please note with something like,
```
XXX portfolio exo state omitted until https://github.com/Agoric/agoric-sdk/issues/10950
```